go.mod

```diff
@@ -132,8 +132,42 @@ require (
 	k8s.io/klog/v2 v2.60.1-0.20220317184644-43cc75f9ae89 // indirect
 	k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf // indirect
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
+	knative.dev/kn-plugin-source-kafka v0.33.0
 	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/kustomize/api v0.10.1 // indirect
 	sigs.k8s.io/kustomize/kyaml v0.13.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
+
+require github.com/maximilien/kn-source-pkg v0.6.3
+
+require (
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/dgraph-io/ristretto v0.0.1 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/ory/viper v1.7.5 // indirect
+	github.com/thediveo/enumflag v0.10.0 // indirect
+	github.com/wavesoftware/go-ensure v1.0.0 // indirect
+)
+
+require (
+	github.com/Shopify/sarama v1.30.1 // indirect
+	github.com/eapache/go-resiliency v1.2.0 // indirect
+	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21 // indirect
+	github.com/eapache/queue v1.1.0 // indirect
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/hashicorp/go-uuid v1.0.2 // indirect
+	github.com/jcmturner/aescts/v2 v2.0.0 // indirect
+	github.com/jcmturner/dnsutils/v2 v2.0.0 // indirect
+	github.com/jcmturner/gofork v1.0.0 // indirect
+	github.com/jcmturner/gokrb5/v8 v8.4.2 // indirect
+	github.com/jcmturner/rpc/v2 v2.0.3 // indirect
+	github.com/kiegroup/kie-tools/packages/kn-plugin-workflow v0.25.0
```

Comment:
Please, if you get your tagged repo for next release.

@ederign @ljmotta

/cc @vyasgun

Reply:
And yes we moved the midstream under `openshift-knative` org. 😬 I hope that's OK.

Reply:
Hi @dsimansk , for this release we will keep the same version for the workflow plugin.